### PR TITLE
add ssh.py

### PR DESCRIPTION
**Pull request by Junefen**
Define the paramiko ssh/sftp for project using.

### CODE REVIEW

File: virtwho/ssh.py [hunk 1]
```diff
@@ -1,3 +1,152 @@
...
+    def put_file(self, local_file, remote_file):
+        """Upload a local file to a remote machine
+        :param local_file: either a file path or a file-like object to be uploaded.
+        :param remote_file: a remote file path where the uploaded file will be placed.
+        """
+        sftp, conn = self._transfer()
+        sftp.put(local_file, remote_file)
+        conn.close()
+
+    def remove_file(self, remote_file_path):
```
Comment (eko999) [inline virtwho/ssh.py:119]: About the argument `remote_file_path`, is it a file or a dir?
If it's a file, we may keep the uniform name as `remote_file`,
If it's dir, we may keep the uniform name as 'remote_dir`.

File: virtwho/ssh.py [hunk 2]
```diff
@@ -1,3 +1,152 @@
-"""
-Define the paramiko ssh/sftp
-"""
+import os
+import paramiko
+from six import BytesIO
```
Comment (eko999) [inline virtwho/ssh.py:3]: It's my fault, there is no BytesIO required for this file.

File: virtwho/ssh.py (hunk 1 above)
Comment (Junefen) [reply virtwho/ssh.py:119]: Oh, will change it, I even forgot added this function and didn't check the name.

File: virtwho/ssh.py (hunk 2 above)
Comment (Junefen) [reply virtwho/ssh.py]: Removed it.

File: virtwho/ssh.py (hunk 1 above)
Comment (Junefen) [reply virtwho/ssh.py]: Changed it to remote_file, thanks.

**Review by hkx303 — APPROVED** [review]
ACK

**Pull request merged**